tests/e2e/Services/GraphQL/MessagingTest.php

```diff
@@ -21,18 +21,18 @@ public function testCreateProviders()
         $providersParams = [
             'Sendgrid' => [
                 'providerId' => ID::unique(),
-                'name' => 'Sengrid1',
+                'name' => 'Sendgrid1',
                 'apiKey' => 'my-apikey',
                 'fromName' => 'Sender Name',
-                'fromEmail' => 'sender-email@my-domain.com',
+                'fromEmail' => 'sender-email@my-domain.example.com',
```

Comment:
you should not use random domains you don't control. (I haven't changed all instances, but I did change this which seemed harmless.) See https://www.rfc-editor.org/rfc/rfc2606.html